docs/core/common/assets/js/api/core/commands-methods/getall.md

```diff
@@ -0,0 +1,18 @@
+## API -- `$e.commands` -- [`$e.commands.getAll()`](../commands.full.md#getAll)
```

Comment:
How is it related to the document components?

Reply:
https://github.com/elementor/elementor/blob/baad70f02b0b9eb83b67473ac0b4558dfb444ddb/docs/core/common/assets/js/api/core/commands.md

its acting like index for reaching the document component from other places.